main.cpp

```diff
@@ -0,0 +1,39 @@
+#include "stages.h"
+#include <cstdlib>
+#include <ctime>
+#include <iostream>
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+int gameLoop(Board &board) {
+	/*
+	winner = 0, empate
+	winner = 1, gano jugador 1
+	winner = 2, gano jugador 2
+	*/
+	int winner = -1;

```

Comment:
#should define a constant for the `-1`, i.e. `static constexpr int kNoWinner = -1;`